person.rb

```diff
@@ -2,31 +2,37 @@
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     super()
     @id = rand(1..100)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rental = []
```

Comment:
- Hey, :wave: also note that the rentals should be plural and not singular. kindly update this.